editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -2242,54 +2246,61 @@ void TileSetAtlasSourceEditor::edit(Ref<TileSet> p_tile_set, TileSetAtlasSource
 	_update_current_tile_data_editor();
 }
 
-void TileSetAtlasSourceEditor::init_source() {
+void TileSetAtlasSourceEditor::init_source(Vector<Ref<TileSetAtlasSource>> atlases) {
```

Comment:
```suggestion
void TileSetAtlasSourceEditor::init_source(const Vector<Ref<TileSetAtlasSource>> &p_atlases) {
```
Prefix arguments with `p_`, also pass by reference, no need to do unnecessary copying even if it only is refcounting